src/pages/home/report/ReportActionCompose.js

```diff
@@ -476,7 +476,7 @@ class ReportActionCompose extends React.Component {
      * @returns {Boolean}
      */
     isEmojiCode(str, pos) {
-        const leftWords = str.slice(0, pos).split(CONST.REGEX.NEW_LINE_OR_WHITE_SPACE);
+        const leftWords = str.slice(0, pos).split(CONST.REGEX.NEW_LINE_OR_WHITE_SPACE_OR_EMOJIS);
```

Comment:
Good catch on [this](https://github.com/Expensify/App/pull/17551#issuecomment-1519077568) one @PauloGasparSv It's a rookie mistake here. Typo In `CONST` it's `NEW_LINE_OR_WHITE_SPACE_OR_EMOJI` and here it's `NEW_LINE_OR_WHITE_SPACE_OR_EMOJIS`.

@dukenv0307 Can you update here also to `NEW_LINE_OR_WHITE_SPACE_OR_EMOJI`

Reply:
So sorry, I just fixed that